### DISCUSSION TITLE
After multiple cycles of deleting families, rerunning cafe5 still reports errors. And some erroes with lowering the lambda value ( 0.0001 )

### DISCUSSION THREAD

**Discussion by MYP0130** [hahnlab/CAFE5#177] [Q&A] [answered]
Hello,
When running according to the command “./bin/cafe5 --infile Orthogroups.GeneCount.cafe.tsv.new --tree cafe_formated.nwk --output_prefix cafe_result3 --cores 12 -k 3 -p --pvalue 0.05 &>k3.log‘’, an error will be reported，as shown in the following figure
<img width="427" alt="image" src="https://github.com/hahnlab/CAFE5/assets/135034863/84fa7574-d448-4a13-888d-4fdf9216658b">
After multiple cycles of deleting families, rerunning cafe5 still reports errors.

Then I tried to lower the lambda value ( 0.0001 )  (./bin/cafe5 --infile Orthogroups.GeneCount.cafe.tsv.new --tree cafe_formated.nwk --output_prefix cafe_result3 --cores 12 -k 3 -p -l 0.0001). Although it can run successfully, the result file is only three, excluding Gamma _ count.txt, etc.
<img width="533" alt="image" src="https://github.com/hahnlab/CAFE5/assets/135034863/8266aeb0-98c5-4e41-936e-636619d57af8">

Can you give me some adivce?
Thanks.

**Comment by geboro** [comment]
+1!
It would be good to list  the limitations of CAFE5 in the documentation...

**Comment by Innocent-L** [comment]
> +1! It would be good to list the limitations of CAFE5 in the documentation...最好在文档中列出 CAFE5 的局限性......

hello, Did you solve the problem? how did you do to solve it？

**Comment by benfulton** [comment, marked as answer]
There are some answers [here](https://github.com/hahnlab/CAFE5/discussions/132#discussioncomment-6152054).
